src/molecules/gallery-button/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { ButtonGroup, IconButton, Tooltip } from "@mui/material";
+import { AccountBox, ViewAgenda } from "@mui/icons-material";
+import React from "react";
+import useStore from "../../ions/store";
+
+const GalleryButton = () => {
+	const setView = useStore(state => state.setView);
+	return (
+		<ButtonGroup>
+			<Tooltip title="Expand">
+				<IconButton
+					aria-label="show more"
+					onClick={() => {
+						setView("profile");
+					}}
+				>
+					<AccountBox />
+				</IconButton>
+			</Tooltip>
+			<Tooltip title="Expand">
+				<IconButton
+					aria-label="show more"
+					onClick={() => {
+						setView("chip");
+					}}
+				>
+					<ViewAgenda />
+				</IconButton>
+			</Tooltip>
```

Comment:
Most Tooltips have the name "Expand". Keep in mind to change this along the way.

Reply:
for got to change that